.github/workflows/tests.yaml

```
@@ -21,15 +20,8 @@ jobs:
     name: "unit-test conda-store-server"
     strategy:
```

Can we (temporarily?) add this to ensure that all CI entries run well in this PR?

```suggestion
    strategy:
      fail-fast: false
```